server/tests/Api.IntegrationTests/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
I believe that to be consistent, we should use the same testing framework - NUnit - in both the unit and integration tests.

Reply:
I first picked Xunit because I was planning on using the theory attribute. Since I now see it's not really needed, I'll revert back to NUnit.